pizza_cutter/des_pizza_cutter/_coadd_slices.py

```diff
@@ -279,7 +282,8 @@ def _build_slice_inputs(
         weights = np.array(weights)
         weights /= np.sum(weights)
 
-    return slices, weights, se_info_to_use
+    possible_slices = slices_to_use
```

Comment:
This list of slices would exclude slices that overlap properly but have non-zero image flags due to a missing PSF model or some such thing.

Reply:
there are two lists: one that is everything that overlapped, and another of things that passed cuts

Reply:
But there is a cut at the top on the image flags in the info struct.

Reply:
It is this extra cut `se_info['image_flags'] == 0`. We are not returning images that don't pass this cut.